src/algebra/gcd_monoid/div.lean

```diff
@@ -59,8 +59,8 @@ begin
   exact mt finset.gcd_eq_zero_iff.1 (λ h, hfz $ h x hx),
 end
 
-theorem gcd_div_gcd_id_eq_one (s : finset ℤ) {x : ℤ} (hx : x ∈ s) (hnz : x ≠ 0) :
-  s.gcd (λ b, b / (s.gcd id)) = 1 :=
+theorem gcd_eq_one_of_div_gcd_id (s : finset ℤ) {x : ℤ} (hx : x ∈ s) (hnz : x ≠ 0) :
```

Comment:
Thanks! I still think the names I suggested [here](https://github.com/leanprover-community/mathlib/pull/16838#discussion_r1014664556) are better though. I can't make sense of the `of` in the names, and think we should either use `coprime` everywhere or use `gcd_eq_one` everywhere. (And if we decide to use `gcd_eq_one`, it should be `gcd_dvd_gcd_id_eq_one` IMO.) @Vierkantor What do you think? Should we make a poll?

Reply:
I agree with @alreadydone to promote consistency and to avoid `_of_` here. I don't have a strong preference though between `gcd_..._eq_one` or `coprime_...` or `..._coprime`.

Reply:
Looking at the library it seems `gcd_div_eq_one` it's better.

Reply:
Which names follow that pattern? The binary version [nat.coprime_div_gcd_div_gcd](https://leanprover-community.github.io/mathlib_docs/data/nat/gcd/basic.html#nat.coprime_div_gcd_div_gcd) includes `div_gcd` in the name.

Reply:
I just mean that there a bunch of `gcd_eq_one`, that is the literal statement.